metanode/partition_op_extent.go

```diff
@@ -454,13 +456,17 @@ func (mp *metaPartition) ObjExtentsList(req *proto.GetExtentsRequest, p *Packet)
 }
 
 // ExtentsTruncate truncates an extent.
-func (mp *metaPartition) ExtentsTruncate(req *ExtentsTruncateReq, p *Packet) (err error) {
+func (mp *metaPartition) ExtentsTruncate(req *ExtentsTruncateReq, p *Packet, remoteAddr string) (err error) {
 	if !proto.IsHot(mp.volType) {
 		err = fmt.Errorf("only support hot vol")
 		p.PacketErrorWithBody(proto.OpErr, []byte(err.Error()))
 		return
 	}
-
+	fileSize := uint64(0)
+	start := time.Now()
+	defer func() {
+		auditlog.LogInodeOp(remoteAddr, mp.GetVolName(), p.GetOpMsg(), req.FullPath, err, time.Since(start).Milliseconds(), req.Inode, fileSize)
```

Comment:
please try add audit log at a common place, like set forbidden switch.

Reply:
It seems that cannot add audit log at a common place, the `full path` field need to be acquired from `*Req` structure.